### PR TITLE
Use hardhat deployments as test fixtures

### PR DESCRIPTION
**Pull request by nkuba**
We have deployment scripts for contracts that should be used in tests, instead of duplicating the deployments of contracts. This PR updates tests to use the contracts deployed with the hardhat scripts.

### CODE REVIEW

File: solidity/test/bridge/Bridge.MovingFunds.test.ts
Comment (pdyraga) [inline solidity/test/bridge/Bridge.MovingFunds.test.ts:1575]: Self-note: check if it is really needed once tests are green.

**Review by pdyraga — COMMENTED** [review]
Tests are failing but the conception looks good to me. Would love @lukasz-zimnoch to also take a look given it's quite a wide change. On my end, once tests are green I am good to merge (assuming the test fix will be easy change).

**Comment by nkuba** [comment]
@pdyraga looks like it's green

File: solidity/test/fixtures/bridge.ts
Comment (pdyraga) [inline solidity/test/fixtures/bridge.ts:53]: It is used only in one place I think so we could also define it in a separate function to do not pollute what's returned from `bridgeFixture()`.

File: solidity/test/bridge/Bridge.MovingFunds.test.ts
Comment (pdyraga) [reply solidity/test/bridge/Bridge.MovingFunds.test.ts:1575]: Yes, it is needed.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**
